CHANGELOG.md

```diff
@@ -37,6 +37,8 @@
 
 ### Fixes
 
+* `[jest-mock]` Replace timestamps with invocationCallOrder
```

Comment:
could you include that it's a breaking change here?

Reply:
@SimenB changes made